frontend/src/metabase/embedding-sdk/theme/MetabaseTheme.ts

```diff
@@ -129,6 +129,9 @@ export type MetabaseComponentTheme = {
 
   /** Data tables **/
   table: {
+    /** Background color of the table header that stays fixed while scrolling. Defaults to `white` if no cell background color is set */
```

Comment:
I'm not sure the default value is entirely correct. Looking at the implementation in `frontend/src/metabase/data-grid/components/DataGrid/DataGrid.tsx`, the default value seems to be derived from either `theme.cell.backgroundColor` or `--mb-color-background` (which derives from `theme.background`).

But maybe saying `white` is simple enough and we don't need to be 100% correct about that.

Reply:
Ok I checked again, we always set the default cell bacgkround color to white 
https://github.com/metabase/metabase/blob/73cedc73b78d485ee155e93a4faaf48782477d59/frontend/src/metabase/embedding-sdk/theme/default-component-theme.ts#L113